src/api-docs/source/includes/_node_rpc_names.md

```diff
@@ -3,9 +3,11 @@
 ## getnameinfo
 
 ```shell--curl
+name='pi';
```

Comment:
lets remove the semicolon here. we leave it out in the other examples as well so the variable assignments work in bash and JS. See https://hsd-dev.org/api-docs/?shell--curl#sendbid for an example

Reply:
(unless you think we should change the style all over the website, might make more sense idk)

Reply:
ah alright, I can do that. I saw a semicolon somewhere, so I assumed that was what it should be.

Reply:
we might just  need an overall consistency check since these docs have been updated by multiple people over several years

Reply:
Yeah, I definitely agree. I'm not really up to that challenge, but I did notice quite a number of inconsistent examples throughout the docs.